packages/devops/scripts/ci/validateBranchName.sh

```diff
@@ -22,6 +23,17 @@ function get_branch_name() {
     echo $branch_name
 }
 
+function check_name_is_not_reserved() {
```

Comment:
@edmofro also sneaked this change in - not strictly related to the scope of this PR, more related to your recent work.

I thought we wouldn't want people accidentally updating the `e2e` instance by pushes, since it should be an auto-generated isntance